SH4RE/SH4RE/FirebaseDB.swift

```diff
@@ -102,3 +103,44 @@ func collectionRead(collectionPath : String, completion: @escaping ((QuerySnapsh
     }
     return
 }
+
+extension UIImage {
```

Comment:
what does this method do? Can we add some comments

Reply:
Just allows us to resize our image for easier storage in firebase cloud storage